### DISCUSSION TITLE
How to install Tailwind CSS custom forms?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#194] [Q&A] [answered]
In short.

**Comment by fcasamento** [comment, marked as answer]
Hi @montasim .

# 1. Install the plugin using npm or Yarn:

###  Using npm
``
npm install @tailwindcss/custom-forms --save-dev
``
###  Using Yarn
``
yarn add @tailwindcss/custom-forms -D
``

# 2. Add the plugin to your tailwind.config.js file

```
// tailwind.config.js
module.exports = {
  // ...
  plugins: [
    require('@tailwindcss/custom-forms'),
  ]
}
```
